tutorials/misc/plot_hack_circle.py

```diff
@@ -0,0 +1,101 @@
+"""
```

Comment:
In the future you might want to instead just add a CircleCI step that does the commands:
```
pip install pytest pytest-cov
pytest mne/viz/tests/test_raw.py
```
or so. You could do it before the docs even get built so it happens faster. But in theory it shouldn't matter as long as what you have here actually fails

Reply:
looks like it's not failing https://app.circleci.com/pipelines/github/mne-tools/mne-python/7552/workflows/eedd2a4b-6040-4c47-9ffe-c05330c26e38/jobs/26615